Jenkinsfile

```diff
@@ -12,7 +12,10 @@ configurationTypes = [
 //     ['OSX_Mojave_12', 'safari'], - These seem to be "too" popular!
 ]
 
+cron_schedule = BRANCH_NAME == "master" ? "0 2 * * *" : ""
```

Comment:
how about ...

`cron_schedule = deployBranches.contains(BRANCH_NAME) ? "H 2 * * *" : ""`

That way we cover ourselves in a bit of load balancing (Any time at 2:xx am), and we also keep up with Line63/66), which means if we build this elsewhere we can monitor each build environment).

Reply:
Makes sense to match the condition in the Full Regression Test. I can't remember where deployBranches comes from at the moment - can you remind me what it is going to contain?

Don't think the H change is going to do much other than start the job at a random time between 2 and 3. It'll most likely still need an agent provisioning at that time of the morning whenever it starts.

Reply:
That's set on line 1. Currently it is just `["master"]` but it could change down the line.

Yeh I just meant we don't care that its exactly 2am, so making it more flexible might help 🤷

Reply:
Ah, blinded by the big array of config types.

If we use H, it'll still be running at the same time every night, just a time that we haven't chosen. Round numbers are nicer in the logs... and if we don't have resource contention, I'd prefer just to leave it at "0 2".